src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -29,9 +29,21 @@
  * </p>
  * <p>
  * Rationale: Putting this comma in make is easier to change the
- * order of the elements or add new elements on the end.
+ * order of the elements or add new elements on the end. Main benefit of a trailing
+ * comma is that when you add new entry to an array, no surrounding lines are changed.
  * </p>
  * <p>
+ * The check allows leaving out the comma at the end if either left
+ * or right curly braces are on the same line as the last element of the array.
```

Comment:
it is not the same as I expected , but ......ok, please make in tests input like:
```
    int[] a1 = new int[] 
    {
        1,
        2
        ,
};

    int[] a2= new int[] 
    {
        1,
        2
        ,};

```

in both cases violations are expected.